src/server/wesnothd/server.cpp

```diff
@@ -590,6 +606,53 @@ void server::dump_stats(const boost::system::error_code& ec)
 	start_dump_stats();
 }
 
+void server::start_dummy_player_updates()
+{
+#if BOOST_VERSION >= 106600
```

Comment:
Didn't we already bump past 1.66 min?

Reply:
Yes, for the TLS PR.

Reply:
Given there's unrelated code that still uses that #if check as well, I'll fix all of them in a separate commit after this PR is merged.